tapir/shifts/tests/test_shift_template_duplicate.py

```diff
@@ -75,8 +75,14 @@ def test_shiftTemplateDuplicateForm_duplicateSingleTemplate(self):
             follow=True,
         )
         self.assertEqual(response.status_code, 200)
-        self.assertEqual(ShiftTemplate.objects.count(), 2)
+        self.assertEqual(2, ShiftTemplate.objects.count())
         for template in ShiftTemplate.objects.all():
+            self.assertEqual(
+                template.start_date.strftime("%Y%m%d"),
+                shift_template.start_date.strftime("%Y%m%d"),
```

Comment:
Is there a reason for converting the dates to strings before comparing? Can't we just do `self.assertEqual(template.start_date,shift_template.start_date)`?